server/liveblog/themes/themes_assets/angular/liveblog-embed/directives.js

```diff
@@ -49,5 +49,31 @@
                     elem.html(attrs.lbTwitterContent);
                 }
             };
-        }]);
+        }])
+        .directive('lbGenericEmbed', ['$timeout', function($timeout) {
+            return {
+                scope: {
+                    item: '='
+                },
+                template: [
+                    '<article class="item--embed" ng-click="item.meta.url">',
```

Comment:
this is faster to load for the client. But later we can setup a gulp/grunt task to compile all the template files with `$templateCache`.